lib/msal-node/src/cache/Storage.ts

```diff
@@ -2,230 +2,101 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import {
-    ICacheStorage,
-    Constants,
-    PersistentCacheKeys,
-    TemporaryCacheKeys,
-} from '@azure/msal-common';
-import { CacheOptions } from '../config/ClientConfiguration';
-import { CACHE } from './../utils/Constants';
-
-const fs = require('fs');
 
-// Cookie life calculation (hours * minutes * seconds * ms)
-const COOKIE_LIFE_MULTIPLIER = 24 * 60 * 60 * 1000;
+import { ICacheStorage } from '@azure/msal-common';
+import { CacheManager } from './CacheManager';
+import { CacheOptions } from '../config/ClientConfiguration';
 
 /**
- * This class implements the cache storage interface for MSAL through browser local or session storage.
- * Cookies are only used if storeAuthStateInCookie is true, and are only used for
- * parameters such as state and nonce, generally.
+ * This class implements Storage for node, reading cache from user specified storage location or an  extension library
  */
 export class Storage implements ICacheStorage {
```

Comment:
Should this be named NodeStorage?

Reply:
apparently "storage" is more intuitive as per other feedback. We can revisit this later.